tests/ExceptionEncoderTest.php

```diff
@@ -101,7 +101,12 @@ protected function assertIncompleteClassStackTrace($exception, $index)
         $encoder = new Understand\UnderstandLaravel5\ExceptionEncoder();
         $stackTraceArray = $encoder->stackTraceToArray($exception->getTrace());
 
-        if (Str::startsWith(phpversion(), ['7.2', '7.3']))
+        if (Str::startsWith(phpversion(), ['7.4', '8.0'])) {
+            // As of PHP 7.4 Exception::getTrace()
+            // no longer contains "args" keys by default
```

Comment:
I don't know enough about the code this is testing to be sure what I'm looking at. With this comment, does that imply that if you do turn on the args capture that this test may not be doing the right thing?

Reply:
Yes exactly https://www.php.net/manual/en/exception.gettrace.php#125278

Reply:
Right, but then what happens if you do enable the flag and run this test? Would checking for an empty array still be a passing test or would it fail? Is it even possible to deal with that?

Reply:
I guess it would fail, but I don't see the point in testing all the possible outcomes based on ini values. If a user enbles the flag it will see the arguments, otherwise it won't.

Reply:
Yeah, sure. If we're confident that args work in >=7.4 then that's fine, really.